tests/v3/covid-19/api_gov.spec.js

```diff
@@ -46,28 +46,25 @@ describe('TESTING /v3/covid-19/gov general', () => {
 });
 
 describe('TESTING /v3/covid-19/gov/canada', () => {
-	it('/v3/covid-19/gov/canada correct amount of provinces', (done) => {
-		chai.request(app)
-			.get('/v3/covid-19/gov/canada')
-			.end((err, res) => {
-				testBasicProperties(err, res, 200, 'array');
-				res.body.length.should.equal(15);
-				done();
-			});
-	});
-
 	it('/v3/covid-19/gov/canada correct fields set', (done) => {
 		chai.request(app)
 			.get('/v3/covid-19/gov/canada')
 			.end((err, res) => {
 				testBasicProperties(err, res, 200, 'array');
+				res.body.length.should.at.greater.or.equal(1);
```

Comment:
incorrect syntax: this is why your tests are failing -- it should be:
`(xxx).should.be.above(yyy);`

Reply:
no idea what i thought when pushing this, must have been quite in a rush